dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -176,4 +176,7 @@ stopAppStartUponListenerException.name=Stop application for unhandled listener e
 stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. When this property is set to true, the application stops starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true. 
 
 redirectToRelativeUrl.name=Send redirect response to a relative URL location
-redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
\ No newline at end of file
+redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location.
+
+allowQueryParamWithNoEqual.name=Allow query parameters with no value
+allowQueryParamWithNoEqual.desc=If the query parameter in a URL contains only the string name instead of the expected name=value format, the server by default returns the value of request.getParameter(name) query as null.  When this property is set to true, if the query parameter in a URL contains only the string name, the server returns an empty string as the value for the request.getParameter(name) query.
```

Comment:
Suggested rewrite:

allowQueryParamWithNoEqual.desc=When this property is set to true, if the query parameter in a URL contains only the string name, the server returns an empty string as the value for the request.getParameter(name) query. When this property is set to false, the server returns null as the value for the request.getParameter(name) query. The default value is false.

Reply:
Thanks for the suggestion @jantley-ibm - I've made this change.